accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -455,7 +456,11 @@ class UsesAsMixin extends OtherSuperclass with Both {} // OK.
 
 This proposal builds on the existing sealed types proposal so the grammar
 includes those changes. The full set of modifiers that can appear before a class
-or mixin are `abstract`, `sealed`, `base`, `interface`, `final`, and `mixin`.
+or mixin declaration are `abstract`, `sealed`, `base`, `interface`, `final`, and `mixin`.
+
+*The modifiers do not apply to other declarations. This includes `enum` declarations.
+**TODO**: Do they apply to inline class declarations?*
```

Comment:
I believe @eernstg updated the inline class proposal to discuss the modifiers there, so we don't need to here.

Reply:
OK, removing it here.